sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -333,7 +333,13 @@ public AutoConfiguredOpenTelemetrySdk build() {
 
     OpenTelemetrySdk openTelemetrySdk = OpenTelemetrySdk.builder().build();
     boolean sdkEnabled =
-        Optional.ofNullable(config.getBoolean("otel.experimental.sdk.enabled")).orElse(true);
+        Optional.ofNullable(config.getBoolean("otel.sdk.disabled"))
+            // default is false, therefore enabled
+            .map(disabled -> !disabled)
+            .orElseGet(
+                () ->
+                    Optional.ofNullable(config.getBoolean("otel.experimental.sdk.enabled"))
+                        .orElse(true));
```

Comment:
Nit: you can use the 2-arg variant of `getBoolean`:
```suggestion
        // default is false, therefore enabled
        !config.getBoolean("otel.sdk.disabled", !config.getBoolean("otel.experimental.sdk.enabled", true));
```
It gets pretty hard to read though, so it might be worth it to extract a variable for the experimental flag.

Reply:
The 2-arg variant returns a primitive boolean, and will not cause an NPE.

Reply:
will use your suggestion